README.md

```diff
@@ -1 +1,74 @@
-# Capstone-project
\ No newline at end of file
+
+# Project Name
+ Capstone-project
+
+## Description
+
+This is my Capstone project that shows how much I learned trough the first Module. Look through some of my work and experience! If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.
+
+## 🖥️ Live Demo
+
```

Comment:
- [ ] Please record a video as requested in the project requirement 

![image](https://user-images.githubusercontent.com/75126481/176749391-35357576-7a08-4ff1-8797-40f9f2f81343.png)